Source/WebCore/page/LocalFrame.h

```diff
@@ -176,7 +176,7 @@ class LocalFrame final : public Frame {
     void resetScript();
 
     bool isRootFrame() const final { return m_rootFrame.ptr() == this; }
-    const LocalFrame& rootFrame() const { return m_rootFrame.get(); }
+    LocalFrame& rootFrame() const { return m_rootFrame.get(); }
```

Comment:
Can we try to keep transitive const-ness by making two root frame accessors?

const LocalFrame& rootFrame() const;
LocalFrame& rootFrame();

The const removal for the type of m_rootFrame is fine.